lib_eio/net.mli

```diff
@@ -27,7 +27,12 @@ type Exn.err += E of error
 val err : error -> exn
 (** [err e] is [Eio.Exn.create (Net e)] *)
 
-(** IP addresses. *)
+(** IP addresses.
+
+  In order to interoperate with the {{:https://opam.ocaml.org/packages/ipaddr/} ipaddr} library we can use it's octet conversion functions.
```

Comment:
```suggestion
  In order to interoperate with the {{:https://ocaml.org/p/ipaddr/} ipaddr} library we can use its octet conversion functions.
```

I think linking to OCaml.org might be preferable and more future proof?

Reply:
Thanks patrick, that makes sense!